rasa/core/schemas/stories.yml

```diff
@@ -45,16 +45,6 @@ mapping:
               action:
                 type: "str"
                 allowempty: False
-          - type: "map"
-            mapping:
-              user:
```

Comment:
we might need to keep this for end-to-end tests

Reply:
depends a bit on what was done here https://github.com/RasaHQ/rasa/issues/6430

Reply:
user messages for story tests (previously e2e tests) are here https://github.com/RasaHQ/rasa/blob/master/rasa/core/schemas/stories.yml#L29

Reply:
Yep, found them, thanks! There will be merge conflicts once `master` is merged into `e2e` either way, so I think it's fine to resolve this then.